src/lib.rs

```diff
@@ -562,7 +562,7 @@ pub fn new_vm() -> RootedThread {
     add_extern_module(&vm, "float_prim", ::vm::primitives::load_float);
     add_extern_module(&vm, "string_prim", ::vm::primitives::load_string);
     add_extern_module(&vm, "char_prim", ::vm::primitives::load_char);
-    add_extern_module(&vm, "array", ::vm::primitives::load_array);
+    add_extern_module(&vm, "std.array", ::vm::primitives::load_array);
```

Comment:
I'm wondering if it would be a good idea to have an `std/array.glu` file, simply for documentation purposes. Would reexport the stuff from `array_prim`.

Also, I'm wondering if it would be better to call these prim modules `std.float.prim`?

Reply:
> I'm wondering if it would be a good idea to have an std/array.glu file, simply for documentation purposes. Would reexport the stuff from array_prim.

True, the `Array` type needs a bunch of functionality such as `Functor`, `Eq` etc anyway so that file will be needed regardless.

> Also, I'm wondering if it would be better to call these prim modules std.float.prim, etc?

That sounds nicer, but I am worried it may be more easier to discover. Perhaps I should just make them impossible to load except by the standard library. Or I can punt that worry until later.

Reply:
We will need some pattern for libraries like those that @endeav0r is working on, so it would probably be a good idea if it the same thing could apply there. Elm uses `*.Native` as a pattern to denote JS code, and requires library authors to specify the modules they expose via their `elm-package.json`.